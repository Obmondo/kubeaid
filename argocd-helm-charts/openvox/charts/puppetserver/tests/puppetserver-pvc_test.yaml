suite: test puppet puppet volume creation
templates:
  - puppetserver-pvc.yaml
release:
  name: puppetserver
  namespace: puppet
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}
  - it: should not create the volume if "puppetserver.persistence.puppet.existingClaim" is non-empty
    set:
      puppetserver.persistence.puppet.existingClaim: dummy
    asserts:
      - hasDocuments:
          count: 0
  - it: should create the volume by default
    asserts:
      - isKind:
          of: PersistentVolumeClaim
