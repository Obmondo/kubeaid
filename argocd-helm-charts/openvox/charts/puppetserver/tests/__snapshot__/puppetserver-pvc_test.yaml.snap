manifest should match snapshot:
  1: |
    apiVersion: v1
    kind: PersistentVolumeClaim
    metadata:
      annotations: null
      labels:
        app.kubernetes.io/component: puppetserver
        app.kubernetes.io/instance: puppetserver
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: puppetserver
        app.kubernetes.io/version: 8.8.0
        helm.sh/chart: puppetserver-10.0.0
      name: puppetserver-puppet-claim
    spec:
      accessModes:
        - ReadWriteOnce
      resources:
        requests:
          storage: 400Mi
